### ISSUE TITLE
Split into two drivers/devices: electricity and gas

### ISSUE THREAD

**Issue by pro-sumer** [koktaildotcom/com.p1.smartmeter#17]
Can you please split the current implementation into two drivers/devices?
* electricity
* gas

The App window is currently overloaded with information (need to scroll to see everything); it would be more clear (to me) if there were two windows (one for electricity and one for gas).

I previously made my own fork to do this, but would like to return to your (newest) App.

**Comment by lucasvdh** [comment]
Personally I prefer the current situation, I only use the values in flows and insights anyway. 

And then there's the case that, if this would be split up into multiple drivers, the Homey would have one more device it needs to run. Don't know exactly how much extra memory this would take up, just that I'd rather use it for something else. 

**TLDR**; if this ends up being implemented, please also keep the current driver for both.

**Comment by koktaildotcom** [comment]
I prefer the current situation because its the device with the capabilities. In theory you could add two devices. One for electricity and one for gas.
